src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -210,4 +209,24 @@ protected Optional<String> buildAppointeeName(SscsCaseData caseData) {
             return Optional.empty();
         }
     }
+
+    private String getNoticeBody(SscsCaseData caseData) {
+        PostHearingReviewType postHearingReviewType = caseData.getPostHearing().getReviewType();
+
+        if (nonNull(postHearingReviewType)) {
+            switch (postHearingReviewType) {
+                case SET_ASIDE:
+                    return caseData.getDocumentGeneration().getBodyContent();
+                case CORRECTION:
+                    return caseData.getDocumentGeneration().getCorrectionBodyContent();
+                case STATEMENT_OF_REASONS:
+                case PERMISSION_TO_APPEAL:
+                case LIBERTY_TO_APPLY:
+                default:
```

Comment:
I'd like to see an exception thrown here. Guards against accidental omission when the remaining post hearing types are implemented. See some example methods in PdfRequestUtil.

Reply:
done